src/app/business/profile/profileCard/profile-card.component.ts

```diff
@@ -147,20 +142,22 @@ export class ProfileCardComponent implements OnInit {
         }else{
             this.isAdministrator = false;
         }
+        this.chartDatas = {
+            labels: ['Unused Capacity', 'Used Capacity'],
+            datasets: [
+                {
+                    data: [this.data.totalFreeCapacity,this.data.totalCapacity-this.data.totalFreeCapacity],
+                    backgroundColor: [
+                        "rgba(224, 224, 224, .5)",
+                        "#438bd3"
+                    ]
+                }]
+        };
     }
 
     index;
     isHover;
 
-    getProfiles() {
-        this.allProfileNameForCheck = [];
-        this.ProfileService.getProfiles().subscribe((res) => {
-            let profiles = res.json();
-            profiles.forEach(item=>{
```

Comment:
what if backend returns duplicate ? better to check and flag/log it rather than show wrong values on UI

Reply:
@rhsakarpos  I think there is some mistake here. Looks like you commented on another line and its showing up somewhere else. Could you check this and let me know ?

Reply:
the comment was for line 157...in this code snippet. 
Question is, what if backend returns duplicate items ? same will show up in UI ?

Reply:
I believe it will. But the premise is that backend API will return only non duplicate values. If the backend API responses are not to be trusted then we have a different issue all together.